packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -130,51 +130,6 @@ const limitBandwidth = (sdp) => {
   return offerSdp;
 };
 
-/**
```

Comment:
Does it make sense to move all our peer connection logic into here instead of the `util.js` file?